Dockerfile

```diff
@@ -18,13 +18,19 @@ RUN set -x \
         nasm \
         libpng-dev \
         python3 \
+        wget \
```

Comment:
Is wget needed?

Reply:
I think it's not needed. It's a residue of testing different "execute as another user" solutions where I have to fetch a binary from another repository.